### DISCUSSION TITLE
i want to set a spritebundle filter to nearest if that is possible cuz my pixel art is blurry

### DISCUSSION THREAD

**Discussion by ZATTYAMOND** [bevyengine/bevy#9118] [Q&A]
i want to set a spritebundle filter to nearest if that is possible cuz my pixel art is blurry help me

**Comment by mockersf** [comment]
You probably want something like 
https://github.com/bevyengine/bevy/blob/8ba9571eedada4f3ff43cdf1402670b7fe7c280d/examples/2d/pixel_perfect.rs#L7-L12
